### PR TITLE
Add cluster visibility user guide

### PR DESCRIPTION
**Pull request by wansir**
Signed-off-by: hongming <talonwan@yunify.com>

### CODE REVIEW

**Comment by Sherlock113** [comment]
/lgtm
@FeynmanZhou The logic and step look good to me. Please merge this pr and I will update a few wording issues. Thanks.

File: content/en/docs/access-control-and-account-management/_index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+layout: "single"
+
+linkTitle: "Accecc Control and Account Management"
+weight: 4500
+
+icon: "/images/docs/docs.svg"
+
+---
+
+## Installing KubeSphere and Kubernetes on Linux
```
Comment (FeynmanZhou) [inline content/en/docs/access-control-and-account-management/_index.md:13]: @wansir @Sherlock113 Add an overview of this chapter in this section, remove the unrelated content,

**Comment by FeynmanZhou** [comment]
@wansir Please resolve the conflicts.

**Comment by Sherlock113** [comment]
/lgtm
@FeynmanZhou Please merge this pr and I will do the rest.

**Comment by FeynmanZhou** [comment]
As I see the [preview page](https://deploy-preview-282--kubesphere-v3.netlify.app/docs/access-control-and-account-management/configuring-authentication/), there is only one doc, how about the rest docs? Are the rest parts still in WIP?

image

**Review by rayzhou2017 — APPROVED** [review]
/lgtm

**Pull request merged**